draft-js-inline-toolbar-plugin/src/components/Toolbar/index.js

```diff
@@ -16,21 +17,53 @@ const getRelativeParent = (element) => {
 };
 
 export default class Toolbar extends React.Component {
-
-  state = {
-    isVisible: false,
-    position: undefined,
-
+  static propTypes = {
     /**
-     * If this is set, the toolbar will render this instead of the regular
-     * structure and will also be shown when the editor loses focus.
-     * @type {Component}
+     * An indents from the left / right window borders
      */
-    overrideContent: undefined
+    toolbarMargin: PropTypes.number
+  };
+
+  static defaultProps = {
+    toolbarMargin: 5
+  };
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      isVisible: false,
+      position: undefined,
+      /**
+       * If this is set, the toolbar will render this instead of the regular
+       * structure and will also be shown when the editor loses focus.
+       * @type {Component}
+       */
+      overrideContent: undefined,
+      /**
+       * We are holding default toolbar width to prevent geometry changing, that
+       * happens very often
+       */
+      width: null,
```

Comment:
Hi, @juliankrispel.
This is important. Browser sometimes makes an attempt to change element's geomentry to fit it in a 'better' way to window. But for us this is not suitable, because our calculations strictly depends on width size. You can see what's going on in the first gif. So we are recording toolbar width after first rendering and then override it sometimes

Reply:
@newsiberian this can also be solved via css - like with `white-space: nowrap` - just needs to be added to the container of the buttons. Give it a shot please, I'd rather not add state if we don't need it